www/getlisted.mako

```diff
@@ -0,0 +1,117 @@
+<%inherit file="_base.mako"/>
+<%def name="title()">Journal Outreach</%def>
+<%def name="description()">Get more journals listed as eligible for the Preregistration Challenge.</%def>
+
+<%def name="navigation()">
+    ${self.navlinks('prereg')}
+</%def>
+
+<%def name="stylesheets()">
+
+    <!-- BEGIN PAGE LEVEL PLUGIN STYLES -->
+    <link href="../static/plugins/fancybox/source/jquery.fancybox.css" rel="stylesheet" />
```

Comment:
What's the purpose of these stylesheets and the embedded styles below? Can these be removed? On first look, they don't seem to have an effect on the page, but I may be missing something.